apps/src/templates/curriculumCatalog/CurriculumCatalogCard.story.jsx

```diff
@@ -14,7 +14,8 @@ const defaultArgs = {
   youngestGrade: 4,
   oldestGrade: 12,
   subjects: ['english_language_arts'],
-  topics: ['cybersecurity']
+  topics: ['cybersecurity'],
+  isTranslated: true
```

Comment:
Just a couple questions on this. 
- How are you checking if something is translated? 
- Are you differentiating translatable vs translated?

Reply:
> Just a couple questions on this.
> 
> * How are you checking if something is translated?

this will come from the `supported_locales` field on the unit model. 

> * Are you differentiating translatable vs translated?

this means translated, not translatable.